plugin/segment/segment.go

```diff
@@ -0,0 +1,123 @@
+package segment
+
+import (
+	"context"
+	"reflect"
+	"time"
+
+	"github.com/bytebase/bytebase/api"
+	"github.com/bytebase/bytebase/store"
+	"github.com/google/uuid"
+	"github.com/segmentio/analytics-go"
+
+	"go.uber.org/zap"
+)
+
+// EventType is the segment track event name.
+type EventType string
+
+var (
+	// InstanceEventType is the track event for instance.
+	InstanceEventType EventType = "instance"
```

Comment:
bb.instance

always prefer namespace as we may allow 3-party integration and they may report their own metrics.